src/game/commands/commands.ts

```diff
@@ -0,0 +1,13 @@
+export const gameCommands = [
+  "/help",
+];
+
+export const playerCommands = [
+  "attack",
+  "drop",
+  "fight",
+  "flee",
+  "move",
+  "run",
+  "walk",
```

Comment:
Adding directions to this list would allow leaving off the move command, retaining existing functionality.

Reply:
Yup, that's already been discussed but will be added later.

Reply:
Allowing directions to be commands in their own right is doable, but it'll add some extra complexity to the parser as the position of a token would influence whether it is X or Y (Command or Direction in this case).

It's probably a good thing to have though, so I'll think about a reasonable way of implementing that.